sklearn/metrics/pairwise.py

```diff
@@ -1288,7 +1288,8 @@ def laplacian_kernel(X, Y=None, gamma=None):
 
     Parameters
     ----------
-    X : ndarray of shape (n_samples_X, n_features)
+    X : ndarray
+        A ndarray of shape (n_samples_X, n_features).
```

Comment:
We keep the shape info at that place. I suggest the following description to be locally consistent
```suggestion
    X : ndarray of shape (n_samples_X, n_features)
        A feature array.
```